AzureCli/readme.md

```diff
@@ -0,0 +1,86 @@
+# Azure CLI
+
+Botservice Extension is an Azure CLI extension to provide support for managing bots and channels on the azure bot service.</br>
+
+## Pre-requisites
+
+Get a [valid azure subscription](https://azure.microsoft.com/en-us/free/)  
+Install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+
+## Installation
+Install the bot service extension as follows -  
+`az extension add --source https://icscratch.blob.core.windows.net/bot-packages/azurebotextension-0.0.1-py2.py3-none-any.whl`
+
+# Managing a Bot
+
+## Creating a Bot
+Login into your azure account via cli :   
+`az login`  
+Ensure that you are using the right subscription for creating your bot -   
+`az account show`  
+Change your subscription , if needed  
+`az account set --subscription <Your-subscription-name>`  
+
+You need an existing resource group to create your bot and related assets. Either use an existing one or create one.
+on the CLI you can create a new resource group as -  
+`az group create -l <resource-group-location> -n <resource-group-name>`
+
+Now create your bot -  
+`az bot create -k webapp -g myResourceGroup -n myBot`  
```

Comment:
When I tried this yesterday, --description was required. Did we fix that? If not, we should include description here.

Reply:
description should be optional. Maybe the package was not udpated when you trried. let me try again and verify